keyboards/acheron/austin/config.h

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2019 Drifting Bunnies, Gondolindrim and PheonixStarr
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x0159
```

Comment:
Surprisingly, this isn't used...

Reply:
Should I remove it?

Reply:
He means that the vendor is not used so you chose well lol

Reply:
^ This. 

I'd ... prefer it be higher, but as long as it's not used .... that's what matters, for now.

Reply:
I put it there because it is not used as a descriptor yet, I think it suffices